plugins/ci/pkg/ci/tfsec.go

```diff
@@ -0,0 +1,90 @@
+package ci
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/fairwindsops/insights-plugins/plugins/ci/pkg/commands"
+	"github.com/fairwindsops/insights-plugins/plugins/ci/pkg/models"
+	"github.com/sirupsen/logrus"
+)
+
+func (ci *CIScan) ProcessTerraformPaths() (models.ReportInfo, error) {
+	logrus.Infof("processing %d Terraform paths", len(ci.config.Terraform.Paths))
+	var reportProperties models.TFSecReportProperties
+	for _, terraformPath := range ci.config.Terraform.Paths {
+		results, err := ci.ProcessTerraformPath(terraformPath)
+		if err != nil {
+			return models.ReportInfo{}, err
+		}
+		if len(results) > 0 {
+			reportProperties.Items = append(reportProperties.Items, results...)
+		}
+	}
+	TFSecVersion, err := commands.Exec("tfsec", "-v")
+	if err != nil {
+		return models.ReportInfo{}, fmt.Errorf("cannot get the version of tfsec: %w", err)
+	}
+	TFSecVersion = strings.TrimPrefix(TFSecVersion, "v")
+	report := models.ReportInfo{
+		Report:   "tfsec",
+		Version:  TFSecVersion,
```

Comment:
I wonder if the report version should reflect `tfsec` tool version or `models.TFSecReportProperties` struct changes

Reply:
I'm currently sending the tfsec version as the report version. We also do this for Trivy. I can see it being useful to vary the report version for internal report changes - is it reasonable to keep the report version aligned with tfsec for now, until we need to vary it?

Reply:
Yes, I think it is reasonable